.eslintrc.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @type {import("eslint").Linter.Config}
```

Comment:
what does this do?

Reply:
guess? 👀 

<img width="644" alt="Screenshot 2022-07-21 at 5 57 16 PM" src="https://user-images.githubusercontent.com/35374649/180213279-ba9726bc-6cd5-4eda-aeb9-93910773b9a0.png">